modules/message-lane/src/lib.rs

```diff
@@ -645,6 +644,16 @@ fn verify_and_decode_messages_proof<Chain: SourceHeaderChain<Fee>, Fee, Dispatch
 	})
 }
 
+/// AccountId of the shared relayer fund account.
+///
+/// This account stores all the fees paid by submitters. Relayers are able to claim these
+/// funds as at their convenience.
+fn relayer_fund_account_id<T: Trait<I>, I: Instance>() -> T::AccountId {
+	use sp_runtime::traits::Convert;
+	let encoded_id = bp_runtime::derive_relayer_fund_account_id(bp_runtime::NO_INSTANCE_ID);
```

Comment:
Shouldn't that be instance aware? Is there a risk if the fund is shared between two instances of the bridge?

Reply:
This account was previously being derived with `NO_INSTANCE_ID` and `MESSAGE_LANE_MODULE_PREFIX` before. This resulted in the same account across different bridge instances, so no behaviour is being changed here